apps/ledger-live-desktop/src/renderer/components/FirebaseFeatureFlags.tsx

```diff
@@ -26,6 +27,17 @@ export const FirebaseFeatureFlagsProvider = ({ children }: Props): JSX.Element =
           return localOverrides[key];
         }
 
+        const envFlags = getEnv("FEATURE_FLAGS") as { [key in FeatureId]?: Feature } | undefined;
+        if (allowOverride && envFlags) {
+          const feature = envFlags[key];
+          if (feature)
+            return {
+              ...feature,
+              overridesRemote: true,
```

Comment:
if it's overriden by env, should remove be false? 🤔

Reply:
It still overrides the remote value, if the name of the variable was different I would have done what you are suggest